### DISCUSSION TITLE
Noob: LS220D

### DISCUSSION THREAD

**Discussion by lammel-hub** [1000001101000/Debian_on_Buffalo#219] [Q&A] [answered]
Hello all,
I am excited to have found this project, as I have my LS220D on my local network, but no longer use it for online sharing and do feel limited by its (lack) of software based functionalities.

I have a few noob questions I hope you can help clear up for me:

1. I currently have a dual HDD RAID setup. Is it possible to keep a RAID functionality using this Debian OS?
2. Is it possible to install emby (or jellyfin, or similar) media server that is accessible from outside the lan (e.g. from the internet or a mobile phone on 5G)
3. Is there any video or screenshots I can review to better understand what the OS looks like when installed on LS220D?
4. Can I re-install the official Buffalo NAS OS (firmware?) in case it all fails?

As you might understand, I have a decent understanding of what this project offers, but I am not fully confident in all technical areas, so hoping for your kind guidance and advice.

**Comment by 1000001101000** [comment]
Good to hear from you!

0. Make sure you backup any important data before installing  anything.
1. Yes
2. I’m not sure, it might work but any feature involving transcoding/etc will be too CPU intensive. It might be better to use this device as an NFS server and have something else be the JellyFin/etc frontend. 
3. Here is what one of mine looks like. The installer provides a very basic OS with just SSH. From there you can install anything you like.
![image](https://github.com/1000001101000/Debian_on_Buffalo/assets/38921757/423d2311-07b3-4b70-bc93-d4ac55d03a17)

4. Yes, The OS is stored on the drives so you can always go back. However, reinstalling the buffalo firmware can be a tricky process and the guides I’ve put together tend to assume wiping the drives before starting.

**Reply by lammel-hub** [reply]
Thank you so much for you swift and consise response.

0-2. Thx, no more questions there :)

3. I am in recovery from MSFT abuse for the past 25 years, so I might have to learn more Linux first, unless it is possible to have a (simple) GUI to manage the NAS. I appreciate the level of control I get when using the terminal, but have not been down that route since the 1990s.
4. If I still try to use debian on it, how would I be able to restore the "Buffalo OS" (or would I make a factory restore as per Buffalo's instructions) in case I need to restore back from Debian?
(I really want to test it, but also be able to revert back in case needed)

Thanks for a great project!

**Comment by 1000001101000** [comment, marked as answer]
The easiest way to test while being able to switch back would be to remove both drives and test with a spare drive. Then you could just put the old drives back to return to the old os.

“cockpit” is a gui that as far as I know is pretty safe to use but somewhat limited in what it does. In my experience webmin and OMV use a shocking amount of system resources and will often break the Debian install by improperly making drive/network changes. As such I don’t recommend using them on these devices.

**Comment by lammel-hub** [comment]
Thank you, now I know how to go about this 🙏
